product_docs/docs/eprs/6.2/index.mdx

```diff
@@ -2,6 +2,11 @@
 title: "Replication Server"
 directoryDefaults:
   description: "EDB Postgres Replication Server Version 6.2 Documentation and release notes. Replication between PostgreSQL databases in single-master or multi-master mode to achieve high availability."
+
+redirects:
```

Comment:
should use `legacyRedirects` for these (paths that exist on the old /edb-docs site, not the new /docs site). For details, see: https://github.com/EnterpriseDB/docs/tree/develop/docs/overviews/legacy-redirects#add_legacy_redirectspy

```suggestion
legacyRedirects:
```